assets/global.js

```diff
@@ -576,15 +576,15 @@ class SliderComponent extends HTMLElement {
     if (this.enableSliderLooping) return;
 
     if (this.isSlideVisible(this.sliderItemsToShow[0]) && this.slider.scrollLeft === 0) {
```

Comment:
The thing I don't get is why is it getting to this part of the code at all ?
Since the buttons are not there on desktop 🤔 I think I'm missing something 😅 
Shouldn't this prevent the event listeners from being setup: 
```javascript
 if (!this.slider || !this.nextButton) return;
```

I guess ideally the right media column would never be a `<slider-component>`. Only the thumbnails get to be a slider if the `desktop layout` is set to `thumbnail carousel` which shouldn't impact the media gallery.

Though that would mean a lot of rework and CSS styling tweak most likely.

Reply:
That's kind of what I was thinking. I feel like there's a more intentional fix we could do, but it could be more involved and this bug is pretty bad.